tests/bloaty_test.cc

```diff
@@ -15,7 +15,10 @@
 #include "test.h"
 
 TEST_F(BloatyTest, EmptyObjectFile) {
-  std::string file = "01-empty.o";
+  if (pe_test_) {
+    GTEST_SKIP_("Object file not yet supported for PE");
+  }
+  std::string file = "01-empty." + ObjExtension();
```

Comment:
I added some minimal changes to existing testcases (and skipped some that are not yet implemented).
Is this a good approach, or would you prefer a completely new test file with PE-specific tests?

Reply:
It's a great question. I've been thinking a lot about Bloaty's tests recently, see: https://github.com/google/bloaty/issues/221

The existing tests are not great: they aren't very robust or reproducible. The problem is that compilers have a lot of discretion when they create object files or binaries. That means that using input like C, Go, etc.doesn't constrain the output enough to have precise tests. We can see that problem here in your PR, where you had to loosen the precision of the tests to get the PE tests to pass. That unfortunately weakens the tests further.

I'm open to input here on the best way to proceed.

Reply:
My suggestion to proceed would be:
- Move all current PE unittests to a new `bloaty_test_pe.cc`
- Here I will create new tests purely for the PE format
- Any overlap between this test and PE files with dwarf info will be something for a later date, when adding PE test files compiled with mingw. This could probably be joined with the current bloaty_test.cc, but we will cross that bridge when we encounter it.